anvil/src/input_handler.rs

```diff
@@ -819,3 +819,15 @@ fn process_keyboard_shortcut(modifiers: ModifiersState, keysym: Keysym) -> Optio
         None
     }
 }
+
+fn detect_terminal() -> String {
+    use std::fs::read_link;
+
+    const SYMLINK: &str = "/usr/bin/x-terminal-emulator";
```

Comment:
This is not a portable way to detect the terminal. Fedora for example does not have this symlink.
I think the best open here would be to read the $TERMINAL environment variable.